spec/rubocop/cop/rails/controller_testing_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Rails::ControllerTesting, :config do
+  context 'Rails 4.2', :rails42, :config do
+    it 'does not add offense when extending ActionController::TestCase' do
+      expect_no_offenses(<<~RUBY)
+        class MyControllerTest < ActionController::TestCase
+        end
+      RUBY
+    end
+  end
+
+  it 'adds offense when extending ActionController::TestCase' do
+    expect_offense(<<~RUBY)
+      class MyControllerTest < ActionController::TestCase
+                               ^^^^^^^^^^^^^^^^^^^^^^^^^^ Use ActionDispatch::IntegrationTest instead. See https://api.rubyonrails.org/classes/ActionController/TestCase.html
+      end
+    RUBY
+
+    expect_correction(<<~RUBY)
+      class MyControllerTest < ActionDispatch::IntegrationTest
+      end
+    RUBY
+  end
+
+  it 'does not add offense when extending ActionDispatch::IntegrationTest' do
+    expect_no_offenses(<<~RUBY)
+      class MyControllerTest < ActionDispatch::IntegrationTest; end
+    RUBY
+  end
```

Comment:
Can you add a test case that does not have a superclass as a test?

Reply:
Added one that tests:

```
    expect_no_offenses(<<~RUBY)
      class MyControllerTest < SuperControllerTest
      end
    RUBY
```

for a custom superclass. Let me know if you meant that, or something else 😄

Reply:
I was worried about `class Foo; end`, but there was no problem.

Reply:
I was worried about `class Foo; end`, but there was no problem :-)